### PR TITLE
Improve output and timing

### PR DESCRIPTION
**Pull request by fkleedorfer**
* Add validation of conversation history at random after receiving a connection message
* print nicer output

To test this bot, either interact with its atoms manually or run the DebugBot on the same node

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,8 +1,5 @@
 bot.name=RandomSimulatorBot
 
-# period of new atom creation, in milliseconds
-atomCreatorBot.period=10000
```
Comment (quasarchimaere) [inline src/main/resources/application.properties:4]: please lets not remove this from here and inject the value directly in the bot.xml, i feel like this is an attribute that can stay in the specific application.properties config

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -40,6 +37,7 @@ botContext.impl=memoryBotContext
 
 keystores.password=changeit
 keystores.directory=${WON_KEYSTORE_DIR}
+#owner.fixedPrivateKeyAlias=
```
Comment (quasarchimaere) [inline src/main/resources/application.properties]: why do we comment this, when in fact we dont do it in any other bot (i know that we result to a good default now, but there is no way of ever figuring out how to add a fixedPrivateKeyAlias if the injection happens deep in our bot framework spring config

**Review by quasarchimaere — CHANGES_REQUESTED** [review]
Also to note `pom.xml` includes a snapshot dependency of won-bot `0.6-SNAPSHOT` this shouldnt not be this way, because otherwise noone will be able to build the bot without building the main project before hand, also there are bouncycastle exclusions within the dependency, i accept that they are commented out but that serves absolutely no purpose a all, please change this

**Comment by fkleedorfer** [comment]
The snapshot dependency cannot be fixed until we have a new release of won-bot.

File: src/main/resources/application.properties (hunk 1 above)
Comment (fkleedorfer) [reply src/main/resources/application.properties:4]: I agree - that's an older commit. Not present in the PR.

File: src/main/resources/application.properties (hunk 2 above)
Comment (fkleedorfer) [reply src/main/resources/application.properties]: I agree. That was commented out to see if the default works now, forgot to uncomment it.

**Pull request merged**